llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -2939,8 +2939,7 @@ LoopVectorizationCostModel::getVectorIntrinsicCost(CallInst *CI,
 
 void InnerLoopVectorizer::fixVectorizedLoop(VPTransformState &State) {
   // Fix widened non-induction PHIs by setting up the PHI operands.
-  if (EnableVPlanNativePath)
-    fixNonInductionPHIs(State);
+  fixNonInductionPHIs(State);
```

Comment:
This patch lowers `VPWidenIntOrFpInductionRecipe` into `VPWidenPhiRecipe`, therefore it removes the restriction that `VPWidenPhiRecipe` is only supported in the VPlan native path.